### PR TITLE
mogrify: tweak descriptions

### PR DESCRIPTION
**Pull request by waldyrious**
This is a follow-up of #1447, which I wasn't able to comment on in time. /cc @wefhy.

----

- [_] ~~The page (if new), does not already exist in the repo.~~

- [_] ~~The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.~~

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/mogrify.md [hunk 1]
```diff
@@ -1,23 +1,24 @@
 # mogrify
 
-> Tool that allows making operations on multiple images ie. resizing, cropping, flipping, adding effects. Replaces the original file.
+> Make operations on multiple images, such as resizing, cropping, flipping, and adding effects.
```
Comment (sbrl) [inline pages/common/mogrify.md]: Make -> perform?

File: pages/common/mogrify.md [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
+- Resize all images starting with "DSC" to 800x600:
 
 `mogrify -resize {{800x600}} {{DSC*}}`
 
 - Convert all png images in the folder to jpg:
 
 `mogrify -format {{jpg}} {{*.png}}`
 
-- Halve the saturation for all image files in the folder:
+- Halve the saturation of all image files in the folder:
```
Comment (sbrl) [inline pages/common/mogrify.md]: `current directory`?

File: pages/common/mogrify.md (hunk 1 above)
Comment (waldyrious) [reply pages/common/mogrify.md]: Yeah, that is better.

File: pages/common/mogrify.md (hunk 2 above)
Comment (waldyrious) [reply pages/common/mogrify.md]: Makes sense, yeah. :+1:

**Pull request merged**